lib/Parser.js

```diff
@@ -65,26 +65,22 @@ export let parseExpression = function(expression, ...contexts) {
         scope = contexts[i];
 
         chain.forEach((item) => {
-            let pos = item.search(functionTest);
+            if (!scope) {
+                return;
+            }
 
-            if (scope) {
-                if (pos > 0) {
-                    let args = item.match(functionTest)[1].split(',').map(item => item.trim());
-                    let scopeChild = scope[item.substring(0, pos)];
```

Comment:
this is missing from the new implementation. Function expressions are no longer properly parsed.